scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncherRebinder.java

```diff
@@ -102,14 +103,14 @@ TaskTerminateNotification makeSureSchedulerIsConnected(TaskTerminateNotification
                 logger.info("On node " + node.getNodeInformation().getName() + " number of active objects found is " +
                             aos.length + " and the first one " + aos[0] + " will be used to send back the task result");
                 return (TaskTerminateNotification) aos[0];
-            } catch (Throwable e) {
+            } catch (Throwable t) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)